examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -61,10 +61,13 @@ you can configure either on a PC or a mobile device.
 ### Bluetooth LE advertising
 
 In this example, to commission the device onto a Matter network, it must be
-discoverable over Bluetooth LE. For security reasons, you must start Bluetooth
-LE advertising manually after powering up the device by pressing:
+discoverable over Bluetooth LE. The Bluetooth LE advertising is automatically
+started after powering up the device, but it may timeout and require re-starting
+by pressing:
 
--   On nRF52840 DK, nRF5340 DK, and nRF21540 DK: **Button 4**.
+-   On nRF52840 DK and nRF5340 DK: **Button 4**.
```

Comment:
Do we want to keep 52840 here while we drop it in workflows?

Reply:
Do you mean 5340? Because 52840 is present in workflows